README.md

```diff
@@ -1,3 +1,101 @@
+
+Read the [Harbor](#harbor) section. Because master is unstable, all development happens in the [rackspace-mk8s-auth branch](https://github.com/rcbops/kubernetes-harbor/tree/rackspace-mk8s-auth) which is always based on a stable release branch. The rackspace-mk8s-auth branch will need to be rebased onto newer Harbor stable branches as we upgrade to newer releases.
+
+The only thing in the master branch is this section of the README just so it's easy to find.
+
+## Dev Env
+
+There are a few things you need to do to get a local dev env on Mac OSX going.
+
+1. Fork the repo.
+
+1. Even though the repo is git@github.com:rcbops/kubernetes-harbor.git it's easier to clone it to a directory structure where Go lang expects everything to be.
+
+    ```bash
+    mkdir $GOPATH/src/github.com/vmware
+    git clone git@github.com:my-github-username/kubernetes-harbor.git $GOPATH/src/github.com/vmware/harbor
+    cd $GOPATH/src/github.com/vmware/harbor
+    git remote add upstream git@github.com:rcbops/kubernetes-harbor.git
+    ```
+
+1. Create a data dir.
+
+    ```bash
+    sudo mkdir /data
+    sudo chown $(whoami):staff /data
+    ```
+
+1. Open your Docker preferences and under File Sharing add dirs `/data` and `/var/log`.
+
+## Run and Configure Kubernetes Auth
+
+1. Fork and clone [kubernetes-auth](https://github.com/rcbops/kubernetes-auth).
+
+1. Run it with the dummy backend in the same network as Harbor.
+
+    ```bash
+    make run NETWORK_NAME_BASE=make_harbor
+    ```
+
+## Run Harbor
+
+This is cribbed from the [compile guide](docs/compile_guide.md). This deploys all Harbor components using your local Docker/Docker Compose. A few changes have been made to [harbor.cfg](make/harbor.cfg) to make this work out of the box.
+
+```bash
+make install GOBUILDIMAGE=golang:1.7.3 COMPILETAG=compile_golangimage CLARITYIMAGE=vmware/harbor-clarity-ui-builder:1.2.7
+```
+
+## Login to Harbor as admin
+
+1. Open [registry.127.0.0.1.nip.io](http://registry.127.0.0.1.nip.io).
+
+1. You can login using the username/password combo `admin/Harbor12345`. Becaue of the way Harbor is designed, the admin user is the one and only user that will use Harbor's own DB for authentication.
+
+## Login to Harbor as a user
+
+1. To login to Harbor as a user, you first need to create a user/token in the kubernetes-auth dummy backend.
+
+1. Work through the [kubernetes-auth example](https://github.com/rcbops/kubernetes-auth#example) to create a user.
+
+1. Open [registry.127.0.0.1.nip.io](http://registry.127.0.0.1.nip.io).
+
+1. You can now login using the token as the password created in step 2. kubernetes-auth only uses the token for auth. The username isn't used at all. In fact, a user could put anything at all into the username field. It will be ignored.
```

Comment:
The auth service will have returned a token. This token will be used as your Harbor password. It's important to note that since the Auth service has no concept of a Harbor username, you can choose any value when inputting your Harbor username - it will be ignored. The only identifier is the token returned by the Auth service